paddle/fluid/platform/mkldnn_helper.h

```diff
@@ -328,6 +328,23 @@ inline mkldnn::memory::format_tag GetMKLDNNFormat(
   return mkldnn::memory::format_tag::undef;
 }
 
+inline mkldnn::memory::format_tag GetPlainMKLDNNFormat(int tensor_rank) {
+  switch (tensor_rank) {
+    case 1:
+      return mkldnn::memory::format_tag::a;
+    case 2:
+      return mkldnn::memory::format_tag::ab;
+    case 3:
+      return mkldnn::memory::format_tag::abc;
+    case 4:
+      return mkldnn::memory::format_tag::abcd;
+    case 5:
+      return mkldnn::memory::format_tag::abcde;
+  }
+
+  return mkldnn::memory::format_tag::abcdef;
```

Comment:
You can make it `default` `switch` statement as well. As always in such _default_ cases I would recommend to check whether the input tensor rank is really what you're assuming to - here 6D. You never know how it may be used in the future. So it would make much faster to spot any errors if there would have been at least some warning or better assertion.